ci/docker_integration_tests.sh

```diff
@@ -1,2 +1,5 @@
 #!/bin/bash
+if [ -n "${FEATURE_FLAG}" ]; then
+  export DOCKER_ENV_OPTS="${DOCKER_ENV_OPTS} --env FEATURE_FLAG=${FEATURE_FLAG}"
+fi
```

Comment:
This should be enough, naming only the env var will use its value by default, and we don't need to check if it's set. If it's not then the container wont have it as well

```suggestion
export DOCKER_ENV_OPTS="${DOCKER_ENV_OPTS} -e FEATURE_FLAG"
```